app/controllers/login_controller.rb

```diff
@@ -23,8 +23,15 @@ def callback
     session[:oauth_response] = JSON.parse(response)
 
     # Redirect the user's browser to the intended home page
-    redirect_to users_home_path
-  end
+    redirect_to users_home_path, success: "You have successfully obtained an access token!"
+
+    rescue RestClient::ExceptionWithResponse
+      if session[:oauth_response]
+        redirect_to users_home_path, danger: "Something Went Wrong. Please Refresh your Access Token and Try Again"
```

Comment:
Does every word in the error message need to be capitalized?

There should be period after "try again", too.